pkg/components/storage.go

```diff
@@ -58,6 +63,18 @@ func startCSIPLugin(cfg *config.MicroshiftConfig, kubeconfigPath string) error {
 			"assets/components/odf-lvm/topolvm-node-securitycontextconstraint.yaml",
 		}
 	)
+
+	// the lvmd file should be located in the same directory as the microshift config to minimize coupling with the
+	// csi plugin.
+	l, err := lvmd.NewLvmdConfigFromFileOrDefault(filepath.Join(filepath.Dir(cfg.ConfigFile), "lvmd.yaml"))
```

Comment:
Renaming `l` to something like `lvmdSettings` here would make it more obvious what it holds.